docs/hooks.md

```diff
@@ -39,6 +39,16 @@ System.constructor.prototype.createContext = function (url) {
 };
 ```
 
+#### createScript(url) -> HTMLScriptElement
+
+When SystemJS loads a module, it creates a `<script>` tag and injects it into the head.
+
+`createScript` allows hooking this script tag creation, will by default implement `return Object.assign(document.createElement('script'), { url, crossOrigin: 'anonymous' })`.
```

Comment:
The default implementation here is missing the charset and async properties that exist in the real implementation. Not sure if that matters though and should be shown in the docs or not.

Reply:
Yeah this is just an example.

Reply:
Although point taken it could be more comprehensive, but the expectation is that users would still call the original hook in line with standard SystemJS hooks.